apps/publish_mdm/forms.py

```diff
@@ -431,3 +432,31 @@ class Meta:
     Organization, TemplateVariable, form=TemplateVariableForm, extra=0
 )
 TemplateVariableFormSet.deletion_widget = CheckboxInput
+
+
+class FleetForm(PlatformFormMixin, forms.ModelForm):
+    class Meta:
+        model = Fleet
+        fields = ["name", "policy", "project"]
+        widgets = {
+            "name": TextInput,
+            "policy": Select,
```

Comment:
I think we want to leave out the `policy` field from the user facing form but still allow it to be specified via the Django admin.
```suggestion
```

The default policy can be provided via settings and/or an environment variable.